.github/workflows/ci.yml

```diff
@@ -272,15 +272,14 @@ jobs:
           BABEL_ENV: test
           BABEL_8_BREAKING: true
           BABEL_TYPES_8_BREAKING: true
+      - name: Prepare artifacts
+        run: |
+          ./scripts/get-artifact-files.sh | tar --null -cvf babel-artifact.tar --files-from=-
       - uses: actions/upload-artifact@v3
         with:
-          name: babel8-test-artifact
-          path: |
-            codemods/*/lib/**/*
-            eslint/*/lib/**/*
-            packages/*/lib/**/*
-            packages/babel-standalone/*.js
-            !**/node_modules/**
+          name: babel8-artifact
+          path: babel-artifact.tar
+          retention-days: 5
```

Comment:
The default retention days is 90, much longer than we need because the artifact is not for download purpose. Here we decrease the time so we can free up some CI storage for other projects.